force-app/main/default/lwc/wireGetRecord/wireGetRecords.js

```diff
@@ -0,0 +1,22 @@
+import { LightningElement, wire, api } from 'lwc';
+import { getRecords } from 'lightning/uiRecordApi';
+import NAME_FIELD from '@salesforce/schema/User.Name';
+import EMAIL_FIELD from '@salesforce/schema/User.Email';
+
+export default class WireGetRecords extends LightningElement {
+    @wire(getRecords, { 
+        records: [
+            { 
+                recordIds: ['005xx000001X83aAAC', '005xx000001X8BdAAK'], 
```

Comment:
hard coded ids should be avoided. It will fail as the same ids will not be available in the deployed org.

Reply:
How would you suggest specifying two unique IDs here using the approach you suggested for the other file?

Reply:
You can wire the Apex method to retrieve user records - https://developer.salesforce.com/docs/component-library/documentation/en/lwc/apex_wire_method